include/odp/api/spec/ipsec.h

```diff
@@ -192,6 +234,33 @@ typedef struct odp_ipsec_outbound_config_t {
 
 } odp_ipsec_outbound_config_t;
 
+/**
+ * IPSEC TEST capability
+ */
+typedef struct odp_ipsec_test_capability_t {
+	/** Support for updating internal parameters of the SA for test */
+	odp_support_t sa_update;
+
+	/** Parameters supported for sa_update */
+	struct {
+		/** Inbound specific internal parameters */
+		struct {
+			/** Highest authenticated sequence number */
+			odp_bool_t antireplay_window_top;
```

Comment:
I wonder if it is convenient to have this kind of array of booleans instead of e.g. returning an array of supported parameter "type" (or parameter ID) values. Maybe @psavol would like to comment.

If also the actual parameter struct for the update call were defined a bit differently, then the test functionality supported by this API could perhaps be extended by ODP implementation specific parameters without an API change.

Reply:
>If also the actual parameter struct for the update call were defined a bit differently,

I am not sure if I understood this correctly. Can you describe the layout you have in mind.

Reply:
Let's keep the capability fields as booleans, but let's also simplify the structure by moving all the fields to the top level. The inbound and outbound substructures do not seem to add any value and the field names have to be unique anyway because the parameter to be modified is specified by the same enum that must have unique corresponding values.

Reply:
Agreed. Made the capability structure to be flat in V6.